src/OpenTracing/Exceptions/InvalidReferenceArgument.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace OpenTracing\Exceptions;
+
+use InvalidArgumentException;
+
+final class InvalidReferenceArgument extends InvalidArgumentException
+{
+    public static function emptyType()
```

Comment:
I would either use distinct classes for these "types" or pass the message directly. Otherwise I don't see a point in having this class at all as these two static methods could just be factory functions that return `InvalidArgumentException`. Also never seen this pattern in PHP.
Also think that methods should always contain a verb in imperative form.